spk/postgresql/src/start-stop-status.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+  
+DAEMON_USER="`echo ${SYNOPKG_PKGNAME} | awk {'print tolower($_)'}`"
+DAEMON_ID="${SYNOPKG_PKGNAME} daemon user"
+DAEMON_USER_SHORT=`echo ${DAEMON_USER} | cut -c 1-8`
+
+DATABASE_DIR="${SYNOPKG_PKGDEST}/share/data"
+
+daemon_status ()
+{
+    ps -efa | grep "postgresql" > /dev/null
```

Comment:
Isn't there a way to test explicit process PID from "postmaster.pid" (you can reuse status check from framework script)? I wonder if such check may just always return true as DSM runs its own embedded postgresql instance.